### PR TITLE
[Doc Improvement][Mobile Android Preview][2612316]

### PR DESCRIPTION
**Pull request by mobajemu-msft**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/549a4d37320a2131dbe1203a6b4d8dd765a9de0c/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e7f99618-2559-4af3-96ff-69033c3d7c89) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/549a4d37320a2131dbe1203a6b4d8dd765a9de0c/msteams-platform%2Fm365-apps%2Foverview.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4b18356e-c4ec-4a4f-8596-9df09016b897) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/549a4d37320a2131dbe1203a6b4d8dd765a9de0c/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 87 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/97cb558c-bac4-4a19-a8e9-26203e3b80c8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/publish.md](https://github.com/MicrosoftDocs/msteams-docs/blob/549a4d37320a2131dbe1203a6b4d8dd765a9de0c/msteams-platform%2Fm365-apps%2Fpublish.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/231c2101-5de7-421a-a21f-9e284a6dd221) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-sdk.md](https://github.com/MicrosoftDocs/msteams-docs/blob/549a4d37320a2131dbe1203a6b4d8dd765a9de0c/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-sdk.md) | 92 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/128f0e3f-5a72-4e70-aa07-88f18dbc45b9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md [hunk 1]
```diff
@@ -26,25 +26,15 @@ You need a Microsoft 365 developer subscription sandbox tenant to set up your de
...
-
- 1. Under Custom apps, turn on the **Interaction with custom app** toggle and save.
-
-    :::image type="content" source="images/teams-admin-enable-sideloading.png" alt-text="The screenshot is an example that enables sideloading for custom apps from the Teams Admin Center":::
-
- 1. Apart from Org-wide app settings, custom app policy settings also allow users to upload custom apps to Teams. For more information, see [manage custom app policies and settings](/microsoftteams/teams-custom-app-policies-and-settings#custom-app-policy-and-settings).
-
- 1. In the Teams admin center, go to **Teams apps** > **Setup policies**, and then select **Global (Org-wide default) policy**.
-
- 1. Turn on **Upload custom apps**, and select **Save**.
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/prerequisites.md:43]: Should these steps be deleted? They're currently in the published doc: https://docs.microsoft.com/en-us/microsoftteams/platform/m365-apps/prerequisites#prepare-a-developer-tenant-for-testing

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 2]
```diff
@@ -209,6 +209,14 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+## Office for Android
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:212]: Whoops, this should be a 3rd level header (H3; use three #s in markdown) section like the ones for other hubs.

**Review by erikadoyle — COMMENTED** [review]
Great start! Added a couple things I noticed and also emailed you screenshots.

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 3]
```diff
@@ -209,6 +209,14 @@ To preview your app running in Office on the web:
...
+
+To view your app running in Office for Android:
+
+1. Launch the Office app and sign in using your dev tenant account.
+1. Select the **More (…)** menu. Your sideloaded app title appears among your installed apps.
```
Comment (sheenamakker) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:217]: This tab is now called "Apps"

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6a423ff4982a555cd6c484b9ab296770f54dbe18/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/110ca727-986d-48aa-ad14-acfab3f0c2dc) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6a423ff4982a555cd6c484b9ab296770f54dbe18/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1ce28347-c0be-4c9f-bf18-6564200b501e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6a423ff4982a555cd6c484b9ab296770f54dbe18/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 87 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/598ccb09-aadf-43d5-99dd-e5d4a4c63003) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/publish.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6a423ff4982a555cd6c484b9ab296770f54dbe18/msteams-platform%2Fm365-apps%2Fpublish.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4567cde0-4282-4097-aa8b-25713430b4e5) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-sdk.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6a423ff4982a555cd6c484b9ab296770f54dbe18/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-sdk.md) | 92 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/cbd1b423-5c0c-4791-a371-935bf1187ff3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ca0f1cd3d9a393d31aea12cf2c8336e125e778d/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 94 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5503b63c-5541-4028-8b20-48da1a741b59) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ca0f1cd3d9a393d31aea12cf2c8336e125e778d/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5061834d-4bc2-4a47-89c0-ee8d174f53c9) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ca0f1cd3d9a393d31aea12cf2c8336e125e778d/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 87 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e6a7e243-db32-4c23-8671-5bdfdef67508) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/publish.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ca0f1cd3d9a393d31aea12cf2c8336e125e778d/msteams-platform%2Fm365-apps%2Fpublish.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/13132351-d934-4196-acbb-56d5adbf715c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-sdk.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1ca0f1cd3d9a393d31aea12cf2c8336e125e778d/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-sdk.md) | 92 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e41c49e1-6a07-47db-a568-bdba5b08765c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/prerequisites.md [hunk 4]
```diff
@@ -84,6 +86,17 @@ To install Office 365 Beta Channel applications in your test environment:
...
+### Mobile
+
+You can preview Teams personal tabs running in Office for Android by joining the [Office Insider Program for Android](https://insider.office.com/en-us/join/Android). 
+
+To install the latest Office Current Channel (Preview) build to your physical Android device or Android emulator:
+
+1. Verify you are using a [supported Android device](https://support.google.com/googleplay/answer/1727131).
+1. [Join the Android best tester program](https://insider.office.com/en-us/join/Android#Join-the-Android-beta-tester-program) to install the Current Channel (Preview) build of the Office app on your Android device.
+1. From your Android device, launch the Office app. Open your profile **(Me) > Settings** and scroll to the bottom of the menu.
+1. Verify you are running version #.#.# (16.0.15620.20000) or higher of Office for Android.
```
Comment (anmaur) [inline msteams-platform/m365-apps/prerequisites.md:98]: This version had a bug. Should we update with current stable version?

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 5]
```diff
@@ -209,6 +209,16 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+### Office for Android
```
Comment (v-ksar) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:212]: Suggest to conform to the branding guidelines for Office, Outlook, and so on. This is a global comment.

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 6]
```diff
@@ -209,6 +209,16 @@ To preview your app running in Office on the web:
...
+
+To view your app running in Office for Android:
+
+1. Launch the Office app and sign in using your dev tenant account.
+1. Select the **Apps** menu. Your sideloaded app title appears among your installed apps.
```
Comment (v-ksar) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:217]: Select the **Apps** icon. Your sideloaded app appears among installed apps.

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 7]
```diff
@@ -233,6 +243,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+1. You can remotely debug your app in Office for Android. From your profile **(Me) > Settings > Allow debugging**, toggle on the option for **Enable remote debugging**.
+1. From your dev machine, open your browser to the DevTools inspection page. For example with Microsoft Edge, open `edge://inspect/#devices` to display a list of debug-enabled Android WebViews. The WebView running your tab application in Office for Android is [placeholder].
```
Comment (v-ksar) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:249]: For example, go to `edge://inspect/#devices` in Microsoft Edge to display a list of debug-enabled Android Webviews.

File: msteams-platform/m365-apps/overview.md [hunk 8]
```diff
@@ -16,10 +16,10 @@ The Teams app platform continues to evolve and expand holistically into the Micr
 
 |          | App manifest element | Teams support |Outlook* support | Office* support | Notes |
 |--|--|--|--|--|--|
-| [**Tabs**](../tabs/what-are-tabs.md) (personal scope)    |`staticTabs`  | Web, Desktop, Mobile | Web (Targeted Release), Desktop (Beta Channel) | Web (Targeted Release)| Channel and group scope not yet supported for Microsoft 365. See [notes](../tabs/how-to/using-teams-client-sdk.md#microsoft-365-support-running-teams-apps-in-office-and-outlook).
+| [**Tabs**](../tabs/what-are-tabs.md) (personal scope)    |`staticTabs`  | Web, Desktop, Mobile (Android) | Web (Targeted Release), Desktop (Beta Channel) | Web (Targeted Release)| Channel and group scope not yet supported for Microsoft 365. See [notes](../tabs/how-to/using-teams-client-sdk.md#microsoft-365-support-running-teams-apps-in-office-and-outlook).
```
Comment (bhups-83) [inline msteams-platform/m365-apps/overview.md:19]: Where is OfficeMobile Android in this table? And why are we reducing staticTabs availability in Teams Mobile to Android only?

**Review by pranavvijayvaran07 — CHANGES_REQUESTED** [review]
Office Add-ins not available for OffciceMobile Android. @bhups-83 can confirm

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 9]
```diff
@@ -233,6 +243,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+1. You can remotely debug your app in Office for Android. From your profile **(Me) > Settings > Allow debugging**, toggle on the option for **Enable remote debugging**.
```
Comment (bhups-83) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:248]: setting name is Enable remote debugging instead of allow debugging.

**Review by pranavvijayvaran07 — CHANGES_REQUESTED** [review]
16.0.15629.20000 and above

File: msteams-platform/tabs/how-to/using-teams-client-sdk.md [hunk 10]
```diff
@@ -32,18 +32,18 @@ The following table lists Teams tabs and dialogs (task modules) capabilities (pu
 
 |Capability | Host support | Notes |
 |-----------|--------------|-------|
-| app | Teams, Outlook, Office | Namespace representing app initialization and lifecycle. |
+| app | Teams, Outlook, Office, Office for Android | Namespace representing app initialization and lifecycle. |
```
Comment (bhups-83) [inline msteams-platform/tabs/how-to/using-teams-client-sdk.md:35]: do we want to be explicit for Outlook platforms (web, OWA, Android and iOS)?

**Review by pranavvijayvaran07 — CHANGES_REQUESTED** [review]
https://docs.microsoft.com/en-us/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?tabs=manifest-teams-toolkit

image

we need to add one more row. Office Mobile d3590ed6-52b3-4102-aeff-aad2292ab01c

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/TOC.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2FTOC.yml) | 98 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46b8ba00-a430-424e-b1dd-5ae29dfa7acb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/design/designing-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fapps-in-teams-meetings%2Fdesign%2Fdesigning-apps-in-meetings.md) | 93 | 15 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0f268ade-9d3b-4c67-854a-6565f787edd4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fapps-in-teams-meetings%2Fenable-and-configure-your-app-for-teams-meetings.md) | 98 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9700e313-4c59-40d4-904d-e64f9a443d7d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fapps-in-teams-meetings%2Fmeeting-app-extensibility.md) | 98 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1172a679-db4b-406d-b96b-5ad3dd2422db) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/bots/how-to/conversations/channel-and-group-conversations.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fbots%2Fhow-to%2Fconversations%2Fchannel-and-group-conversations.md) | 94 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/53042caf-9e65-49ec-a9e5-cd45be638dd8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/concepts/build-and-test/shared-channels.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshared-channels.md) | 93 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f08b733c-6c7a-4544-9137-e2d4aa8bdf4a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/feedback.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Ffeedback.md) | 89 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/68c4f84c-f126-4ff4-b88c-2008d7755481) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/get-started/glossary.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fget-started%2Fglossary.md) | 94 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/781ae315-7dbb-4d56-be0f-89b79fa51944) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/extend-m365-teams-personal-tab.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fm365-apps%2Fextend-m365-teams-personal-tab.md) | 93 | 24 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/38885226-98cf-4a49-a365-9b3cf4d3a9e0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fm365-apps%2Foverview.md) | 94 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/16786503-8d4a-4114-8099-e2ebe280b6b0) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/prerequisites.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fm365-apps%2Fprerequisites.md) | 87 | 23 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/209add3a-1eb3-417f-87b6-b28abb0af45b) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/m365-apps/publish.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fm365-apps%2Fpublish.md) | 90 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a027cacc-5502-41ce-8aa1-24eac608e892) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 90 | 26 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ffe43da0-9f33-4d0a-8c40-048c6777f22f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 95 | 19 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/06582fd9-cc56-44bc-be1c-c1385f4124c2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/access-teams-context.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Ftabs%2Fhow-to%2Faccess-teams-context.md) | 91 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9d3d8714-e526-48f9-97c5-2085fb68a2fa) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/tabs/how-to/using-teams-client-sdk.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Ftabs%2Fhow-to%2Fusing-teams-client-sdk.md) | 92 | 16 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5498196-1a4b-430e-8072-8d0ef9941953) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/task-modules-and-cards/cards/cards-format.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Ftask-modules-and-cards%2Fcards%2Fcards-format.md) | 90 | 17 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ab61f57d-679b-41ce-8ba5-4fc7deddd87a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2374924a6e8e3f41b3cec54504dffdec30a08327/msteams-platform%2Fwhats-new.md) | 91 | 35 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9fa14a96-5afe-49c6-92d4-96cf83d4c02c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 11]
```diff
@@ -233,6 +243,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
```
[the remark this answers is not in the document]
Comment (sheenamakker) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:246]: We shouldn't include Outlook right now, they will be code complete end September.

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 9 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md]: *Enable remote debugging* toggle is under the header for "Allow debugging" in settings menu

File: msteams-platform/m365-apps/overview.md (hunk 8 above)
Comment (erikadoyle) [reply msteams-platform/m365-apps/overview.md:19]: Good catch!

File: msteams-platform/m365-apps/overview.md [hunk 12]
```diff
@@ -16,10 +16,10 @@ The Teams app platform continues to evolve and expand holistically into the Micr
...
 | [**Message extensions**](../messaging-extensions/what-are-messaging-extensions.md) (search-based)| `composeExtensions` | Web, Desktop, Mobile| Web (Targeted Release), Desktop (Beta Channel)| - |Action-based not yet supported for Microsoft 365. See [notes](extend-m365-teams-message-extension.md#preview-your-message-extension-in-outlook). |
 | [**Graph connectors**](/graph/connecting-external-content-connectors-overview)| `graphConnector` | Web, Desktop, Mobile| Web, Desktop | Web| See [notes](#graph-connectors)
-| [**Office Add-ins**](/office/dev/add-ins/develop/json-manifest-overview) (preview) | `extensions` | - | Web, Desktop | - | Only available in [devPreview](../resources/schema/manifest-schema-dev-preview.md) manifest version. See [notes](#office-add-ins-preview).|
+| [**Office Add-ins**](/office/dev/add-ins/develop/json-manifest-overview) (preview) | `extensions` | - | Web, Desktop, Mobile (Android) | - | Only available in [devPreview](../resources/schema/manifest-schema-dev-preview.md) manifest version. See [notes](#office-add-ins-preview).|
```
Comment (erikadoyle) [inline msteams-platform/m365-apps/overview.md:22]: Whoops, Android support is not available for Office Add-ins (using Teams apps preview manifest)

**Comment by mobajemu-msft** [comment]
the content of this Pr has been shifted to PR [#6935](https://github.com/MicrosoftDocs/msteams-docs/pull/6935)

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 13]
```diff
@@ -209,6 +209,16 @@ To preview your app running in Office on the web:
 
     :::image type="content" source="images/office-web-more-apps.png" alt-text="Click on the 'More apps' option on the side bar of office.com to see your installed personal tabs":::
 
+### Office for Android
+
+To view your app running in Office for Android:
+
+1. Launch the Office app and sign in using your dev tenant account.
```
Comment (harshatms) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:216]: Shall we increment sequence numbers?

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md [hunk 14]
```diff
@@ -209,6 +209,16 @@ To preview your app running in Office on the web:
...
+1. Select the **Apps** icon. Your sideloaded app appears among installed apps.
```
Comment (harshatms) [inline msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:217]: Do we need to put a note about ensuring, user should join beta and app version should be 16.0.15629.20000 and above?

File: msteams-platform/m365-apps/prerequisites.md [hunk 15]
```diff
@@ -84,6 +86,19 @@ To install Office 365 Beta Channel applications in your test environment:
...
     If it's not listed, install [Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/) to your test environment.
 
+### Mobile
+
+You can preview Teams personal tabs running in Office for Android by joining the [Office Insider Program for Android](https://insider.office.com/en-us/join/Android). 
+
+To install the latest Office Current Channel (Preview) build to your physical Android device or Android emulator:
+
+1. Verify you are using a [supported Android device](https://support.google.com/googleplay/answer/1727131).
+1. [Join the Android best tester program](https://insider.office.com/en-us/join/Android#Join-the-Android-beta-tester-program) to install the Current Channel (Preview) build of the Office app on your Android device.
```
Comment (harshatms) [inline msteams-platform/m365-apps/prerequisites.md:96]: best -> beta

File: msteams-platform/m365-apps/prerequisites.md [hunk 16]
```diff
@@ -84,6 +86,19 @@ To install Office 365 Beta Channel applications in your test environment:
...
+1. From your Android device, launch the Office app. Open your profile **(Me) > Settings** and scroll to the bottom of the menu.
+1. Verify you are running version 1.0.1 (16.0.15627.20006) or higher of Office for Android.
```
Comment (harshatms) [inline msteams-platform/m365-apps/prerequisites.md:98]: 16.0.15629.20000 and higher

File: msteams-platform/resources/schema/manifest-schema.md [hunk 17]
```diff
@@ -782,7 +782,7 @@ Enables your app in non-standard channels. If your app supports a team scope and
 > [!NOTE]
 >
 > * If your app supports a team scope, it functions in the standard channels regardless of the values that are defined in this property.
-> * Your app can account for the unique properties of each of the channel types to function properly. To enable your tab for private and shared channels, see [retrieve context in private channels](~/tabs/how-to/access-teams-context.md#retrieve-context-in-private-channels) and [retrieve context in shared channels](~/tabs/how-to/access-teams-context.md#retrieve-context-in-microsoft-teams-connect-shared-channels).
+> * Your app can account for the unique properties of each of the channel types to function properly. To enable your tab for private and shared channels, see [retrieve context in private channels](~/tabs/how-to/access-teams-context.md#retrieve-context-in-private-channels) and [get context in shared channels](../../tabs/how-to/access-teams-context.md#get-context-in-shared-channels)
```
Comment (harshatms) [inline msteams-platform/resources/schema/manifest-schema.md:785]: Shall we mention min manifest version: 1.13

File: msteams-platform/tabs/how-to/using-teams-client-sdk.md (hunk 10 above)
Comment (erikadoyle) [reply msteams-platform/tabs/how-to/using-teams-client-sdk.md:35]: Yes, we will add iOS upon preview

File: msteams-platform/resources/schema/manifest-schema.md (hunk 17 above)
Comment (erikadoyle) [reply msteams-platform/resources/schema/manifest-schema.md:785]: Min manifest version 1.13 is covered [here](https://review.docs.microsoft.com/en-us/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?branch=pr-en-us-6846&tabs=manifest-teams-toolkit#update-the-app-manifest) and [here](https://review.docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?branch=pr-en-us-6846&tabs=javascript%2Cmanifest-teams-toolkit)

**Comment by mobajemu-msft** [comment]
> https://docs.microsoft.com/en-us/microsoftteams/platform/m365-apps/extend-m365-teams-personal-tab?tabs=manifest-teams-toolkit
> 
> image
> 
> we need to add one more row. Office Mobile d3590ed6-52b3-4102-aeff-aad2292ab01c

resolved

File: msteams-platform/m365-apps/extend-m365-teams-personal-tab.md (hunk 14 above)
Comment (mobajemu-msft) [reply msteams-platform/m365-apps/extend-m365-teams-personal-tab.md:217]: added a note to the new commit

**Comment by erikadoyle** [comment]
@mobajemu-msft let's go ahead and close this now obsolete PR, as the review has moved to https://github.com/MicrosoftDocs/msteams-docs/pull/6935

**Pull request closed without merging**